packages/core/src/components/icon/icon.scss

```diff
@@ -1,11 +1,9 @@
 @use '../../helpers';
+@use '../../mixins';
 
 @mixin Icon() {
-  $size: helpers.space(3);
-
   .ods-icon {
+    @include mixins.square(helpers.space(3));
```

Comment:
Not sure about this.

For example, it would not allow to use when size has to be calculated, or in case of something like `rem` or `em` should be used (which currently it would not support but still).

Reply:
Fair enough, let's leave as is for now.